### PR TITLE
FIX: ISXB-1127 composite incorrectly returning IsPressed

### PR DESCRIPTION
**Pull request by smnwttbr**
### Description

[ISXB-1127](https://jira.unity3d.com/browse/ISXB-1127) is a regression caused by the fix for [ISXB-746](https://jira.unity3d.com/browse/ISXB-746). It was decided to revert this fix, as it is a corner case with an acceptable workaround, and the regression is more serious.

Before releasing this fix, the original customer of ISXB-746 should be notified that the fix has been reverted.

### Testing status & QA

Local tests passed.

### Overall Product Risks

Low risk.

- Complexity: Low 
- Halo Effect: Small

### Comments to reviewers

_Please describe any additional information such as what to focus on, or historical info for the reviewers._

### Checklist

Before review:

- [ ] Changelog entry added.
    - Explains the change in `Changed`, `Fixed`, `Added` sections.
    - For API change contains an example snippet and/or migration example.
    - JIRA ticket linked, example ([case %<ID>%](https://issuetracker.unity3d.com/product/unity/issues/guid/<ID>)). If it is a private issue, just add the case ID without a link.
    - Jira port for the next release set as "Resolved".
- [ ] Tests added/changed, if applicable.
    - Functional tests `Area_CanDoX`, `Area_CanDoX_EvenIfYIsTheCase`, `Area_WhenIDoX_AndYHappens_ThisIsTheResult`.
    - Performance tests.
    - Integration tests.
- [ ] Docs for new/changed API's.
    - Xmldoc cross references are set correctly.
    - Added explanation how the API works.
    - Usage code examples added.
    - The manual is updated, if needed.

During merge:

- [ ] Commit message for squash-merge is prefixed with one of the list:
    - `NEW: ___`.
    - `FIX: ___`.
    - `DOCS: ___`.
    - `CHANGE: ___`.
    - `RELEASE: 1.1.0-preview.3`.

After merge:

- [ ] Create forward/backward port if needed. If you are blocked from creating a forward port now please add a task to ISX-1444.

### CODE REVIEW

**Comment by unity-cla-assistant** [comment]
[CLA assistant check](https://cla-assistant.cds.internal.unity3d.com/Unity-Technologies/InputSystem?pullRequest=2035) <br/>All committers have signed the CLA.

**Comment by Pauliusd01** [comment]
I've let the bug reporter know

**Review by kahyongs — APPROVED** [review]
Regression. Reverting.

**Review by Pauliusd01 — APPROVED** [review]
LGTM, checked original repro steps with keyboard and a couple of mainstream controllers in Playmode and Player as well as sample scenes that seemed loosely related. Let me know if there's anything else that needs checking though

File: Packages/com.unity.inputsystem/CHANGELOG.md [hunk 1]
```diff
@@ -12,6 +12,7 @@ however, it has to be formatted properly to pass verification tests.
 
 ### Fixed
 - Fixed an issue causing the Action context menu to not show on right click when right clicking an action in the Input Action Editor [ISXB-1134](https://issuetracker.unity3d.com/product/unity/issues/guid/ISXB-1134).
+- Reverted changes from 0ddd534d8 (ISXB-746) which introduced a regression [ISXB-1127](https://issuetracker.unity3d.com/product/unity/issues/guid/ISXB-1127)
```
Comment (ekcoh) [inline Packages/com.unity.inputsystem/CHANGELOG.md]: Minor, missing period at end.

**Review by ekcoh — APPROVED** [review]

**Pull request merged**